Buildings/Experimental/DHC/EnergyTransferStations/Cooling/Examples/DirectControlled.mo

```diff
@@ -0,0 +1,159 @@
+within Buildings.Experimental.DHC.EnergyTransferStations.Cooling.Examples;
+model DirectControlled "Example model for direct cooling energy transfer station 
+  with in-building pumping and controlled district return temperature"
+  extends Modelica.Icons.Example;
+  package Medium=Buildings.Media.Water
+    "Water medium";
+  parameter Modelica.Units.SI.HeatFlowRate Q_flow_nominal=-18000
+    "Nominal cooling load (negative for cooling)"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mBui_flow_nominal=-Q_flow_nominal/(cp*(18-7))
+    "Nominal mass flow rate of building cooling supply"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.SpecificHeatCapacity cp=Medium.specificHeatCapacityCp(
+    Medium.setState_pTX(
+      Medium.p_default,
+      Medium.T_default,
+      Medium.X_default))
+    "Default specific heat capacity of medium";
+  Buildings.Experimental.DHC.EnergyTransferStations.Cooling.DirectControlled
+    cooETS(
+    mDis_flow_nominal=0.5,
+    mBui_flow_nominal=mBui_flow_nominal,
+    mByp_flow_nominal=0.01,
```

Comment:
That looks wrong. At the minimum, it should be some fraction of the design flow rate rather than just `0.01`.